src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -11,11 +11,11 @@ library GPv2Encoding {
     ///
     /// The memory layout for an order is very important for optimizations for
     /// recovering the order address. Specifically, all the signed order
-    /// parameters appear in contiguous memory (`sellToken` to
-    /// `partiallyFillable`). This allows the memory reserved for decoding an
-    /// order to also be used for hashing, providing some added effeciency.
+    /// parameters appear in contiguous memory as the first 9 fields from
+    /// `sellToken` to `partiallyFillable`. This allows the memory reserved for
+    /// decoding an order to also be used for computing the order digest during
+    /// the decoding process for some added effeciency.
```

Comment:
What was the reason again for not nesting two structs (have a Settlement struct that contains an Order struct with all the signable fields)? This would make the signing boundaries more explicit.

Reply:
ah, just saw you explanation on #117